framework/Models/PostProcessors/Validation.py

```diff
@@ -192,71 +156,61 @@ def _handleInput(self, paramInput):
         self.features = child.value
       elif child.getName() == 'Targets':
         self.targets = child.value
-    if 'static' not in self.model.dataType and self.pivotParameter is None:
+    if 'static' not in self.dataType and self.pivotParameter is None:
       self.raiseAnError(IOError, "The validation algorithm '{}' is a dynamic model ONLY but no <pivotParameter> node has been inputted".format(self._type))
     if not self.features:
       self.raiseAnError(IOError, "XML node 'Features' is required but not provided")
     elif len(self.features) != len(self.targets):
       self.raiseAnError(IOError, 'The number of variables found in XML node "Features" is not equal the number of variables found in XML node "Targets"')
 
-  def collectOutput(self, finishedJob, output):
+  def initialize(self, runInfo, inputs, initDict):
     """
-      Function to place all of the computed data into the output object
-      @ In, finishedJob, JobHandler External or Internal instance, A JobHandler
-        object that is in charge of running this post-processor
-      @ In, output, dataObjects, A reference to an object where we want
-        to place our computed results
+      Method to initialize the DataMining pp.
+      @ In, runInfo, dict, dictionary of run info (e.g. working dir, etc)
```

Comment:
@wangcj05 Shouldn't the docstring be updated? validation PP instead of DataMining PP?

Reply:
Yes, go ahead and update it.